Traceback (most recent call last):
  File "F:\mygit\python\w01_exchange\run.py", line 22, in main
    0 / 0
ZeroDivisionError: division by zero
